src/Keboola/DbExtractor/Extractor/Firebird.php

```diff
@@ -119,19 +175,32 @@ public function simpleQuery(array $table, array $columns = array()): string
     {
         if (count($columns) > 0) {
             $query = sprintf(
-                'SELECT %s FROM %s',
+                'SELECT %s%s FROM %s',
```

Comment:
aha, this shocked me, thought it was a typo:).  maybe better to have `%s %s` and remove the space from `getIncrementalLimitAddon`?